src/node_crypto.cc

```diff
@@ -2590,10 +2591,15 @@ void CipherBase::Init(const char* cipher_type,
                                1,
                                key,
                                iv);
+  CHECK_NE(key_len, 0);
 
   ctx_.reset(EVP_CIPHER_CTX_new());
   const bool encrypt = (kind_ == kCipher);
-  EVP_CipherInit_ex(ctx_.get(), cipher, nullptr, nullptr, nullptr, encrypt);
+  if (1 != EVP_CipherInit_ex(ctx_.get(), cipher, nullptr,
```

Comment:
Small style nit, but I was curious why you would do this over the regular `EVP_CipherInit_ex(...) != 1`? I find it a bit more readable, but would love to know why you picked this over it.